app/models/push_event.js

```diff
@@ -13,8 +13,9 @@ export default class PushEvent extends BaseModel {
       '_id',
       'created_at',
       'updated_at',
+      'notification_channel_id',
```

Comment:
in `PushSubscription` we use `platform` field. Can we have the filed named as `platform` here as well?